components/builder-api/src/server/token.rs

```diff
@@ -0,0 +1,115 @@
+use super::AppState;
+use crate::server::error::Error;
+use builder_core::{access_token::AccessToken,
+                   privilege::FeatureFlags};
+use diesel::result::{DatabaseErrorKind,
+                     Error as DieselError};
+use habitat_builder_db::models::{account::{Account,
+                                           AccountToken,
+                                           NewAccount,
+                                           NewAccountToken},
+                                 channel::{Channel,
+                                           CreateChannel},
+                                 origin::{NewOrigin,
+                                          Origin},
+                                 package::PackageVisibility};
+use std::{fs::{self,
+               File},
+          io::Write};
+
+const BLDR_USER_NAME: &str = "chef-platform";
+const BLDR_USER_EMAIL: &str = "chef-platform@progress.com";
```

Comment:
could we make the username and email configurable in the `provision` config? I can see a use case for using this auto provisioning outside of just chef 360. For example, you might have a on-prem customer who would like to do some automated package uploading before logging in via oauth. They could use a valid github user and email here, create the initial api key and then login with the same account via oauth later. This has come up with customers in the past.